main.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python3
+"""
+Example of using Rerun to log and visualize the out of grounded dino + segment-anything.
+
+See: [segment_anything](https://github.com/IDEA-Research/Grounded-Segment-Anything).
+
+Can be used to test mask-generation on one or more images, as well as videos. Images can be local file-paths
+or remote urls. Videos must be local file-paths. Can use multiple prompts.
+"""
+
+
+import argparse
+import logging
+import rerun as rr
+import torch
+import cv2
+from pathlib import Path
+from models import CONFIG_PATH, MODEL_URLS, get_downloaded_model_path
+from models import load_grounding_model, create_sam, load_image, image_to_tensor
+from models import get_grounding_output, run_segmentation, resize_img
+from segment_anything import SamPredictor
+from segment_anything.modeling import Sam
+from groundingdino.models import GroundingDINO
+
+
+def log_images_segmentation(args, model: GroundingDINO, predictor: Sam):
+    for n, image_uri in enumerate(args.images):
+        rr.set_time_sequence("image", n)
```

Comment:
```suggestion
        rr.set_time_sequence("image", n)
        predictor.set_image(image)
```
Set the image once per image